sstables/sstable_directory.hh

```diff
@@ -250,7 +250,7 @@ public:
     // until all shards agree it can be deleted.
     //
     // This function only solves the second problem for now.
-    static future<> delete_atomically(std::vector<shared_sstable> ssts);
```

Comment:
The second and third paragraph are outdated. 

But of course it's unrelated to your series.

nit: with_pending_deletion_log